netbout-web/src/test/java/com/netbout/rest/TkFriendTest.java

```diff
@@ -52,14 +53,14 @@
     public void buildsPngImage() throws Exception {
         final Base base = new MkBase();
         final String alias = "test";
-        final URN urn = new URN("urn:test:1");
-        final User user = base.user(urn);
+        final String urn = "urn:test:1";
+        final User user = base.user(new URN(urn));
```

Comment:
@zshamrock I know it was like this, but we should leave the code better than it was before :)